### ISSUE TITLE
Connect problem 

### ISSUE THREAD

**Issue by J4CKED** [Telegram-FOSS-Team/Telegram-FOSS#769]
Since yesterday, the telegram has not worked (connecting...). The website does not work either. Status on the page indicates the problem
https://downforeveryoneorjustme.com/telegram
Does anyone know what's going on?

**Comment by VasiliyMooduckovich** [comment]
yeah i got the same issue i guess the telegram team done something with their API or something, i switched to other client works fine
